scout/server/blueprints/alignviewers/templates/alignviewers/igv_viewer.html

```diff
@@ -28,7 +28,7 @@
             src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.2/jquery-ui.min.js"></script>
 
     <!-- IGV JS-->
-     <script type="text/javascript" src="https://igv.org/web/release/2.2.11/dist/igv.min.js"></script>    
+     <script type="text/javascript" src="https://igv.org/web/release/2.2.11/dist/igv.min.js"></script>
```

Comment:
Seems there is now a 2.2.17. Any reason not to track the current? 

There are also pre-builts there of the new ones as well for download. That would make it a little bit more independent, but more bulk in the repo. As long as there are no issues with the dl, that is preferable.

Reply:
I'm adding version 2.2.17

Reply:
:+1: They also have a 2.3-pre-release. Lets not go there yet, but keep track of it!

Reply:
I've installed the updated branch on stage and it's working. I'll merge then. Thanks for the review!